addons/mrp/wizard/mrp_batch_produce.py

```diff
@@ -116,7 +116,6 @@ def _production_text_to_object(self, mark_done=False):
         for production in reversed(productions):
             production.qty_producing = production.product_uom_qty
             production.set_qty_producing()
-            production.move_raw_ids.picked = True
```

Comment:
Shouldn't it be only for moves that should be manually consumed?

Reply:
Hello @clesgow ,

No, here it marked all the move_raw_ids to True, which is why it needs to be removed.
Additionally, this PR has been retargeted to version 17.2 now, so could you please review it again?

Thanks for the suggestion!